velox/docs/monthly-updates/december-2021.rst

```diff
@@ -0,0 +1,38 @@
+********************
+December 2021 Update
+********************
+
+Documentation
+-------------
+
+* Add :doc:`Aggregation-related optimizations <../develop/aggregations>` guide.
+* Add missing documentation for :func:`json_extract_scalar` function.
+* Add documentation for missing functions in :doc:`Mathematical Functions <../functions/math>` and :doc:`Comparison Functions <../functions/comparison>`.
+* Update :doc:`How to add an aggregate function? <../develop/aggregate-functions>` guide to reflect changes to the function registry.
+* Update :doc:`How to add a scalar function? <../develop/scalar-functions>` guide to describe how to write functions with complex types inputs and results.
+
+
+Core Library
+------------
+
+* Add support for partition key aliases to Hive Connector.
+* Add support for conversions between Velox string and Arrow string.
+* Add support for 'timestamp with timezone' in :func:`to_unixtime` function.
```

Comment:
Since this is a Presto function, this update should go into "Presto Functions" section.